src/binding/actix/server_request.rs

```diff
@@ -156,23 +156,18 @@ mod tests {
     use super::*;
     use actix_web::test;
 
+    use crate::test::*;
     use crate::{EventBuilder, EventBuilderV10};
     use serde_json::json;
-
     #[actix_rt::test]
     async fn test_request() {
-        let expected = EventBuilderV10::new()
-            .id("0001")
-            .ty("example.test")
-            .source("http://localhost/")
-            .extension("someint", "10")
-            .build()
-            .unwrap();
+        let mut expected = v10::minimal();
```

Comment:
Alternatively, we could do this, the explicitness of it I kinda like:
```suggestion
        let mut expected = fixtures::v10::minimal();
```

Reply:
Either way, we don't need the "pub use".